R/fit.R

```diff
@@ -15,9 +15,6 @@ CmdStanFit <- R6::R6Class(
       self$runset$num_procs()
     },
     print = function(variables = NULL, ..., digits = 2, max_rows = 10) {
-      if (!length(self$output_files(include_failed = FALSE))) {
-        stop("Fitting failed. Unable to print.", call. = FALSE)
-      }
```

Comment:
Need to remove this so it doesn't error if output files aren't available. Otherwise we can't use this with the output from read_cmdstan_csv(). 

However, this doesn't risk `print()` failing to catch an important error because the same error will be caught by the `metadata()` method that is called internally by `print()`. `metadata()` is smarter about throwing the error because it only throws the error if the files are missing AND `metadata()` hasn't already been read in.